src/model.jl

```diff
@@ -180,11 +186,31 @@ function _make_sets(case::Case, storage::Union{Storage,Nothing})::Sets
                                * "Did you forget to linearize_gencost?")
     num_segments = convert(Int, maximum(case.gencost[:, NCOST])) - 1
     segment_idx = 1:num_segments
+    # Demand flexibility, additional info for input->bus conversion
+    if demand_flexibility.enabled
+        if demand_flexibility.input_granularity == "BUS"
+            # assume each flexible bus can go up/dn, so only use the Dataframe for up
+            flexible_bus_str = names(demand_flexibility.flex_amt_up)[2:end]
```

Comment:
Might want to include a quick comment for why the index starts at 2 here, I'm guessing the CSV is expected to have a line with column names?

Reply:
This follows from my previous (probably bad convention). We could probably make this cleaner throughout the whole function by indicating that the first column is an index when we first load the CSVs.